user/enterprise/operations-manual.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Travis CI Enterprise Operations manual
+layout: en_enterprise
+
+---
+
+This document is meant as a guideline to follow in case your Travis CI Enterprise production installation becomes unresponsive or unavailable.
+
+If this happens, we'd like to ask you to follow the steps outlined in this document first before you get in touch with us to save important data and collect as much evidence as possible. This will help us to provide you the best support possible and focus completely on resolving the problem.
+
+## First: Don't panic
+
+The first thing you should do is: keep calm. We're here to help and we're confident that we can bring it back online together with you!
+
+## Prerequisites
+
+Please make sure that you have root access on all machines Travis CI Enterprise is running on. This is crucial because we will ask you to perform operations where you either need to be root or you need to have the appropriate `sudo` privileges. If you don't have direct root privileges, please make sure that a person who has is available.
+
+## Coordinating with Support
+
+We are here to help troubleshoot and fix any problems that may arise with your Travis CI Enterprise installation. For efficiency, though, we would request you minimize making changes to your setup that are not directly recommended by support, or document these steps and notify support accordingly.
+
+## Terminology
```

Comment:
I'd probably skip this when reading a troubleshooting doc so I'd ditch the title. And I'd reword to something like

> Throughout this document we'll be using the following terms to refer to the two components of your Travis CI Enterprise installation:

- `Platform machine`: The virtual machine that runs most of the Travis web components. This is the machine your domain is pointing to.
- `Worker machine`: The worker machine(s) that run your builds.

In some parts of the docs we call these hosts? And in some machines? Can we be consistent

Reply:
I like the definitions - people do use 'host' and 'platform' OR 'worker' interchangeably, and often expect master/slave diction too... however, perhaps this section can be moved somewhere where it's ~less likely to be skipped~ more helpful?